git-open

```diff
@@ -77,30 +97,40 @@ IFS='/' pathargs=($urlpath)
 
 if (( is_issue )); then
   # For issues, take the numbers and preprend 'issues/'
-  providerBranchRef="issues/${branch//[^0-9]/}"
+  providerBranchRef="/issues/${branch//[^0-9]/}"
 else
   # Make # and % characters url friendly
   #   github.com/paulirish/git-open/pull/24
   branch=${branch//%/%25} branch=${branch//#/%23}
-  providerBranchRef="tree/$branch"
+  providerBranchRef="/tree/$branch"
 fi
 
 if [[ "$domain" == 'bitbucket.org' ]]; then
   # Bitbucket, see https://github.com/paulirish/git-open/issues/80 for why ?at is needed.
-  providerBranchRef="src?at=$branch"
+  providerBranchRef="/src?at=$branch"
 elif [[ ${pathargs[0]} == 'scm' ]]; then
   # Bitbucket server, which starts with 'scm'
   # Replace the first element, 'scm', with 'projects'. Keep the first argument, the string 'repos', and finally the rest of the arguments.
   pathargs=('projects' ${pathargs[1]} 'repos' "${pathargs[@]:2}")
   IFS='/' urlpath="${pathargs[*]}"
-  providerBranchRef="browse?at=$branch"
+  providerBranchRef="/browse?at=$branch"
+elif [[ "${#pathargs[@]}" -ge "2" && ${pathargs[${#pathargs[@]} - 2]} == "_git" ]]; then
+  # Visual Studio Team Services and Team Foundation Server
```

Comment:
can you add a comment saying VSTS always will have /_git/ as the second to last segment in the url path?

a bit easier to read than parsing L117 ;)

Reply:
Definitely.  Added a comment to clarify what was going on here.